### ISSUE TITLE
Unable to open selector image interface (file manager) 

### ISSUE THREAD

**Issue by danjde** [hvdwolf/jExifToolGUI#78]
Hi Devs!
I've just installed on Ubuntu 18.04, jexiftoolgui-1.6.3.deb, but seems imposible to list image file from "open Image"

Java version:
```
openjdk version "1.8.0_252"
OpenJDK Runtime Environment (build 1.8.0_252-8u252-b09-1~18.04-b09)
OpenJDK 64-Bit Server VM (build 25.252-b09, mixed mode)
```

Error log:
```
(java:12397): GLib-GObject-WARNING **: 18:03:22.712: invalid cast from 'GtkToplevelAccessible' to 'JawToplevel'
18:03:36.159 [AWT-EventQueue-1] INFO  org.hvdw.jexiftoolgui.mainScreen - Selected: Load Images
Exception in thread "AWT-EventQueue-1" java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key stfio.loadimages
	at java.util.ResourceBundle.getObject(ResourceBundle.java:450)
	at java.util.ResourceBundle.getString(ResourceBundle.java:407)
	at org.hvdw.jexiftoolgui.controllers.StandardFileIO.getFileNamesAwt(StandardFileIO.java:220)
	at org.hvdw.jexiftoolgui.mainScreen.loadImages(mainScreen.java:497)
	at org.hvdw.jexiftoolgui.mainScreen$SpecialMenuActionListener.actionPerformed(mainScreen.java:2209)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.AbstractButton.doClick(AbstractButton.java:376)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:842)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:886)
	at java.awt.Component.processMouseEvent(Component.java:6539)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6304)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4535)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:760)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:733)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:730)
	at org.GNOME.Accessibility.AtkWrapper$6.dispatchEvent(AtkWrapper.java:715)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

Thanks!

**Comment by hvdwolf** [comment]
Hi,

The error `Exception in thread "AWT-EventQueue-1" java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key stfio.loadimages` means that it can't find the translation for the key string `stfio.loadimages`. I just unpacked the 1.6.3 and indeed there is a strange character in the default properties file.
It should say: `stfio.loadimages = Load Image(s)... (or select other file filter)`
but in the default fallback file it says: `stfio.loadimages\u00a0=\u00a0Load Image(s)... (or select other file filter)`
I have no idea how that happened.
And it also mentions for the next one `stfio.loadfolder\u00a0= Locate the image folder ...`

What is the language you are using on your system?
As your system language is not supported it defaults to the "fallback" default file (also English).
I suggest to restart jExifToolGUI and please go into  Preferences ->Lanuguage. 
Please set it to `en_US - English`, restart and try again.

**Comment by danjde** [comment]
Bingo!
It really depend from the language setting: in fact setting as default lang  "en_US - English" all seems woks fine.
're good! ;-)


Thanks!

**Comment by hvdwolf** [comment]
I will release a bugfix 1.6.3.1 this afternoon/evening.

(I am not ready with my 1.7 yet)

**Comment by hvdwolf** [comment]
fixed in 1.6.3.1
